uefi-test-runner/src/main.rs

```diff
@@ -73,7 +73,7 @@ fn check_screenshot(bt: &BootServices, name: &str) {
 
         // Set a large timeout to avoid problems with Travis
         let mut io_mode = *serial.io_mode();
-        io_mode.timeout = 10_000_000;
+        io_mode.raw.Timeout = 10_000_000;
```

Comment:
Same here, it would make sense to have a `set_timeout` accessor or similar on this type.

Reply:
I will add them all!